src/rpc/blockchain.cpp

```diff
@@ -1480,6 +1480,7 @@ UniValue getchaintxstats(const JSONRPCRequest& request)
             "{\n"
             "  \"time\": xxxxx,        (numeric) The timestamp for the statistics in UNIX format.\n"
             "  \"txcount\": xxxxx,     (numeric) The total number of transactions in the chain up to that point.\n"
+            "  \"block_count\": xxxxx, (numeric) The block height that ends the window.\n"
```

Comment:
This seems to be a confusing name; I would associate `block_count` with the size of the window over which `txrate` is computed. To denote the block height that ends the window, I would use `height`.